src/Shared/TokenGenerator.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Security.Cryptography;
+
+namespace Aspire.Hosting;
+
+internal static class TokenGenerator
+{
+    public static string GetToken()
+    {
+        // Generate a 128-bit entropy token 
+        var tokenBytes = GenerateEntropyToken(16); // 16 bytes = 128 bits 
```

Comment:
Even though the code is defined below, I find named literal arguments easier to read.

```suggestion
        var tokenBytes = GenerateEntropyToken(size: 16); // 16 bytes = 128 bits 
```